eng/Versions.props

```diff
@@ -216,9 +216,7 @@
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
-    <GoogleProtobufPackageVersion>3.8.0</GoogleProtobufPackageVersion>
-    <GrpcAspNetCoreServerPackageVersion>0.1.21-pre1</GrpcAspNetCoreServerPackageVersion>
-    <GrpcToolsPackageVersion>1.21.0</GrpcToolsPackageVersion>
+    <GrpcAspNetCorePackageVersion>0.1.22-pre1</GrpcAspNetCorePackageVersion>
```

Comment:
Note. Need to publish a version of this metapackage.

Reply:
We'll update this to 0.1.22-pre2 when it gets published. this version is currently published to the nightly feed so we this PR is unblocked.